### DISCUSSION TITLE
Publication Format

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#372] [Ideas]
I know we're a long way off from having anything to publish, but Tim Thompson's recent [talk at the LD4 conference](https://2022ld4conferenceonlinkedda.sched.com/event/13scj/beyond-spreadsheets-metadata-crosswalks-as-structured-documentation) about [DITA](https://en.wikipedia.org/wiki/Darwin_Information_Typing_Architecture#Maps ) got me thinking about how we want to publish our mapping. Could DITA be the way to go?

Of note, someone in the presentation said the RDA Toolkit uses DITA.

**Comment by AdamSchiff** [comment]
Yes, it is true that the official RDA toolkit uses DITA.  I don't know how difficult it is to learn, but there are only a couple of people at LC who have learned it to input the LC-PCC policy statements.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________
From: Crystal Clements ***@***.***>
Sent: Wednesday, July 20, 2022 4:49 PM
To: uwlib-cams/MARC2RDA ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [uwlib-cams/MARC2RDA] Publication Format (Discussion #372)


I know we're a long way off from having anything to publish, but Tim Thompson's recent talk at the LD4 conference<https://2022ld4conferenceonlinkedda.sched.com/event/13scj/beyond-spreadsheets-metadata-crosswalks-as-structured-documentation> about DITA<https://en.wikipedia.org/wiki/Darwin_Information_Typing_Architecture#Maps> got me thinking about how we want to publish our mapping. Could DITA be the way to go?

Of note, someone in the presentation said the RDA Toolkit uses DITA.

**Reply by CECSpecialistI** [reply]
Seems like we could make use of it for a number of projects around here. Any interest in learning @briesenberg07 or @gerontakos ? I'm intrigued and would be willing to spend some time on it

**Comment by briesenberg07** [comment]
I too was intrigued by Tim Thompson's talk, because I hadn't ever really considered DITA. 

I'm interested in assessing whether it would be useful for publishing MARC2RDA data (and/or other data which we create and reuse--I'm thinking about human-readable Sinopia templates, UW digital collections MAPs, etc.). I don't know whether or not it would, what it can be used for, how to use it, etc.

**Comment by GordonDunsire** [comment]
As Thompson's presentation says, DITA is well-supported by commercial applications. For RDA Toolkit, those are Alfresco content management system, and Oxygen XML editor for offline working. There isn't much to learn; the application handles markup and assignment of IDs for content re-use. DITA is an essential part of the Toolkit infrastructure because of the high degree of modularisation, content re-use at fine levels of granularity, and integration with RDF/XML data from the RDA Registry at least three times per year.

One route to explore for this project is to publish in the Community Resources area of RDA Toolkit (via NARDAC or EURIG). This is very much under development, with issues of access (for free or fee) yet to be determined. There is also the RDA Registry ...

btw, I disagree with the impression given of a crosswalk continuum; we make the distinction between 'map' and 'alignment' in the Registry (ie. RDF and not RDF). See [A Reconsideration of Mapping in a Semantic World](https://dcpapers.dublincore.org/pubs/article/download/3622/1848.pdf)

**Comment by gerontakos** [comment]
I don't think a good case was made for using DITA. Perhaps my view is based on misunderstandings of Thompson's talk, but I don't see this as a solution to the problem of representing an alignment (thinking in RDA terms, as we know how to represent a "mapping": we use OWL, right?). There are some nice results in the Yale "crosswalk" like consistent ways of representing conditions -- but really those conditions are just Boolean operators strung together. We can do that in a spreadsheet too. So I see DITA as a nice supplement -- supplementary documentation -- but not a solution for our problem.

I favor an approach that is a "direct" representation of the instruments in ontology 1 brought into some relation with the instruments in ontology 2. In other words, I favor a tool made specifically for the alignment endeavor. DITA is not a tool built for representing alignments, although it can be used that way. The way I see it, after watching Thompson's talk, the Yale method represents the target ontology using DITA (i.e. in a structure of DITA "topics") and the source ontology gets mapped to the target-ontlogy-as-DITA. I would be very displeased to attempt to represent RDA as DITA topics and display it as a DITA map (I'm sure we could take another approach). The specification index for MARC-to-Linked Art is quite small, making this approach possible -- yet the effort to create the mapping still seemed enormous. In addition, I would like to note that Yale not only mapped MARC within a context of DITA and the Linked Art extension, but also within the context of the LUX discovery interface, so I would say this representation of the mapping can be considered extremely "indirect."

An example of a direct representation would be something like XeOML (see Stellato 2004: https://www.academia.edu/3055845/XeOML_An_XML_based_extensible_Ontology_Mapping). That's a tool made for the job. It's an old recommendation and I don't think it's been used much. It's more complex than we need while, at the same time, may have trouble representing our "conditions." Of course, we could use widely adopted tools like OWL and SPARQL when it comes time to publish. No matter what, someone is going to have to comb through the spreadsheets and produce the representation (our spreadsheet's cell values for-the-most-part are not structured).

There are other efforts out there: Euzenat's expressive alignment format (quite complicated; see https://hal.inria.fr/file/index/docid/822892/filename/kweb-2210.pdf), I think Scharffe has something in SEKT but I haven't taken the time to figure that out (https://www.sekt-project.com/resources/). I still haven't looked at the solutions by the Samvera people -- but I don't even know where to look for that. We could also build our own; in that case, at least the alignment would be represented as RDF (presumably). If creating-our-own too wild, we could base it on something like the now-twenty-year-old MAFRA (https://www.dei.isep.ipp.pt/~nsilva/R&D/Publications/2003/MAFRA%20-%20An%20Ontology%20MApping%20FRAmework%20for%20the%20Semantic%20Web(OntoBIS%272003).pdf). There's got to be many more options...
